content/technology/solutions/altinn-platform/receipt/_index.en.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Receipt
+description: The Receipt component in Altinn Platform presents a receipt for apps.
```

Comment:
I'm a little unsure about the placement of the documentation here. My hunch is that `content/technology/solutions/altinn-platform/receipt/_index.en.md` is more of a "technical" documentation about the receipt component, and not so easily accessible for app-developers which I think use the section under `content/app/development/x` .

Should we consider moving the overriding of texts to for instance https://docs.altinn.studio/app/development/ux/texts/ Or maybe a link to this page from the text section is enough?

Reply:
Thats probably a good point :) 

Moving it to texts seems like a good approach to me. 

- [ ] Move to texts location